release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1183,19 +1183,48 @@ As part of the continued deprecation of the Operator package manifest format, th
 * Previously, the documentation and `oc explain` help text did not convey that the `buildArgs` field in the `BuildConfig` object does not support the `valueFrom` field of its underlying Kubernetes `EnvVar` type. As a result, users believed it was supported and tried to use it. The current release updates the documentation and help text, so it is more apparent that the `BuildConfig` object's `buildArgs` field does not support the `valueFrom` field.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1956826[*BZ#1956826*])
 
-
 * When builds interact with image registries, such as pulling base images, intermittent communications issues can produce build failures. The current release increases the number of retries to these interactions. Now, OpenShift builds are more resilient when they encounter intermittent communication issues with image registries.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1937535[*BZ#1937535*])
 
-
 *Cloud Compute*
 
+* Previously, a proxy update caused a full cluster configuration update, including API server restart, during continuous integration (CI) runs. Consequently, some clusters in the Machine API Operator would time out because of unexpected API server outages. This update separates proxy tests and adds postconditions so that clusters in the Machine API Operator become stable again during CI runs. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1913341[*BZ#1913341*])
 
+* Previously, deleting a machine stuck in `Insufficient disk space on datastore` took longer than expected because there was no distinction between various vCenter task types. With this update, the machine-controller deletion procedure now checks the vCenter task type, and no longer blocks the deletion of the machine-controller. As a result, the machine-controller is deleted quickly.  (link:https://bugzilla.redhat.com/show_bug.cgi?id=1918101[*BZ#1918101*])
```

Comment:
```suggestion
* Previously, deleting a machine stuck in `Insufficient disk space on datastore` took longer than expected because there was no distinction between various vCenter task types. With this update, the machine controller deletion procedure checks the vCenter task type, and no longer blocks the deletion of the machine controller. As a result, the machine controller is deleted quickly.  (link:https://bugzilla.redhat.com/show_bug.cgi?id=1918101[*BZ#1918101*])
```